### ISSUE TITLE
Multi-person real-time inference

### ISSUE THREAD

**Issue by Ared521** [fabro66/GAST-Net-3DPoseEstimation#69]
Hello, thank you for your great work. Has the real-time attitude estimation of multiple people been completed?

**Comment by sk-zhang** [comment]
Hi, have you already realized it?

**Comment by Ared521** [comment]
> Hi, have you already realized it?

No, i think 2d sequence to 3d pose is diffcult in multi-person.

**Comment by sk-zhang** [comment]
> > Hi, have you already realized it?
> 
> No, i think 2d sequence to 3d pose is diffcult in multi-person.

Thanks for the reply.
